src/core/core.tooltip.js

```diff
@@ -782,6 +785,12 @@ module.exports = function(Chart) {
 				me._active = [];
 			} else {
 				me._active = me._chart.getElementsAtEventForMode(e, options.mode, options);
+				if (me._lastActive.length > 0 && me._active.length < 1) {
```

Comment:
I think this check could be a little simpler `if (me._lastActive.length && !me._active.length)`. Same idea with the one below. For minification it might also make sense to store `me._lastActive.length` and `me._active.length` in local variables